protos/spice.proto

```diff
@@ -78,13 +78,8 @@ message SimResult {
 }
 // # Simulator Options 
 // Global, cross-analysis settings. 
-// FIXME: all of these should probably be "explicitly optional" as `google.protobuf.DoubleValue`s or similar. 
 message SimOptions {
-  double temp = 1;
-  double tnom = 2;
-  double gmin = 3;
-  double iabstol = 4;
-  double reltol = 5;
+  google.protobuf.Struct opts = 1;
```

Comment:
So the `Value` type used for the values of each field is a handy shortcut here. 
https://protobuf.dev/reference/protobuf/google.protobuf/#value

But I think, for similar reasons to why `Prefixed` was created in the first place, that the values here should probably be `vlsir.utils.ParamValue`. 
https://github.com/Vlsir/Vlsir/blob/e7f5f91a0a4e14469cd33c6a5c3b9b1b321a6058/protos/utils.proto#L51

Moving the keys to arbitrary strings rather than structured fields seems fine. These could be a protobuf `map`; in most cases we have instead ordered them and used the `repeated`/ list-like construct of a thing with both the key and value. Prime example: `vlsir.utils.Param`:
https://github.com/Vlsir/Vlsir/blob/e7f5f91a0a4e14469cd33c6a5c3b9b1b321a6058/protos/utils.proto#L65

So, if ya buy that, this would become something like:

```protobuf 
message SimOptions {
  repeated vlsir.utils.Param options = 1;
}
```

Reply:
This is a good solution that will allow me to leave `build.rs` alone, I might tweak it to add a name field and add a `bool` as an `oneof` attribute, just to cover all uses and make it work like `Param`.

It seems that the definition of `google.protobuf.Struct`:

```proto
message Struct {
  // Unordered map of dynamically typed values.
  map<string, Value> fields = 1;
}
```

breaks `prost`'s ability to serialize a `Struct`. Losing the `map<string, Value>` only really means for downstream is that you can use:

```python
myoption1 = h.sim.Option(3)
myoption2 = h.sim.Option(4,name="your-name-here")
```
And not:

```python
myoption3 = h.sim.Option(your_name_here=5)
```

Which is not even a loss.